### PR TITLE
doc: Editing Domain Section

### PR DESCRIPTION
**Pull request by CamilleLetavernier**
- Document how to execute commands in custom Model Service implementations

### CODE REVIEW

**Comment by CamilleLetavernier** [comment]
Note: we keep mentioning "Editing Domain" for Model Manager command execution. "Editing Domain" was the correct term to use in the Model Server, but this concept doesn't really exist in the Model Manager framework (we don't use the term anywhere). Should we change this name, to avoid confusion?

**Comment by eneufeld** [comment]
We used Editing Domain in our initial split of functionality.
Now we use context for the same thing in my understanding.

File: content/documentation/editingDomain/content.md [hunk 1]
```diff
@@ -6,4 +6,178 @@ weight = 160
...
 +++
 
-TODO Describe the model manager on a very high level, but mostly how it is integrated for a language with the model hub and which of the Model Hub capabilities are then covered generically
+#### General principles
+
+Edition of models is handled by the ModelManager, by executing Commands on one or several CommandStacks. The ModelManager is not directly exposed by the ModelHub API, so it is not (easily) possible to execute arbitrary commands on arbitrary models. Instead, the ModelManager is passed to ModelServiceContributions, which can then use it in their custom ModelService implementation to execute commands. This way, clients do not have to deal with Commands directly, but simply interact with the custom API.
+
+#### Model Service implementation
+
+As seen in the ModelHub section, the ModelServiceContribution is the entry point for language-specific contributions. Upon initialization of the ModelHub, each contribution will get access to the ModelManager, and should typically forward it to their ModelService implementation.
```
Comment (eneufeld) [inline content/documentation/editingDomain/content.md]: we should link the ModelHub section here

Comment (eneufeld) [inline content/documentation/editingDomain/content.md]: `language-specific contributions` shouldn't we keep this more generic as `model-specific contributions` ?

**Review by eneufeld — COMMENTED** [review]
I think this section should have a section describing contexts and stackids and their difference

**Comment by CamilleLetavernier** [comment]
> I think this section should have a section describing contexts and stackids and their difference

I added a paragraph, but it's a bit difficult to explain without proper examples. AFAIK, we don't use context or stack ids in the Coffee Editor. Moreover, the general Model Manager documentation should already cover these topics in details, I think.

**Review by eneufeld — APPROVED** [review]
I really like the short summary of the stackId and context.
As soon as the modelmanager is opensourced we can reference its documentation from here.

**Pull request merged**